blocksuite/framework/std/src/gfx/interactivity/manager.ts

```
@@ -881,217 +881,173 @@ export class InteractivityManager extends GfxExtension {
-          ext.onResizeStart?.({
-            elements,
-            handle,
-            handlePos,
-            handleSign,
-          });
+      suggested.sort((a, b) => (a.priority ?? 0) - (b.priority ?? 0));
+      return last(suggested) ?? { scaleX, scaleY };
+    },
+    onResizeStart: ({ handleSign, handlePos, data }) => {
+      this.activeInteraction$.value = { type: 'resize', elements };
+      extensionHandlers.forEach(ext =>
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)